server/sonar-server/src/main/java/org/sonar/server/es/EsClient.java

```diff
@@ -193,6 +202,17 @@ public Client nativeClient() {
     return nativeClient;
   }
 
+  /**
+   * Checks whether there is any document in any mentioned type.
+   */
+  public boolean isEmpty(IndexTypeId... indexType) {
+    return count(indexType) <= 0;
+  }
+
+  public long count(IndexTypeId... indexType) {
```

Comment:
In my opinion the varargs parameter gives more flexibility to the caller. Why do you want it to be removed?